Server/src/models/GraphsModel.ts

```diff
@@ -120,9 +152,10 @@ export class GraphsModel {
      * @param axisUnits 
      * Axis unit types: string[]
      */
-    private async sendSavedGraphToDatabase(accountId: number, name: string, datasetIds: number[], datasetColors: string[], datasetShapes: string[], datasetHiddenStatus: boolean[],
-        axisVariable: string[], axisLog: boolean[], axisZoomStart: number[], axisZoomEnd: number[], axisUnits: string[]): Promise<string> {
-        let newGraph = new Savedgraphs();
+    private async sendSavedGraphToDatabase(accountId: number, name: string, datasetIds: number[], datasetColors: string[], datasetShapes: string[],

```

Comment:
Instead of passing several params here, you can pass the original obj. This can be refactored to use the preserve object design pattern - https://refactoring.guru/preserve-whole-object.